addons/udes_stock/tests/common.py

```diff
@@ -30,26 +95,73 @@ def setUpClass(cls):
             })
         cls.test_locations = cls.test_location_01 + cls.test_location_02
 
-        # Products
+        out_zone = cls.picking_type_pick.default_location_dest_id
+        cls.test_output_location_01 = Location.create({
+            'name': "Test output location 01",
+            'barcode': "LTESTOUT01",
+            'location_id': out_zone.id})
 
-        # Untracked
-        cls.apple = cls.create_product('Apple')
-        cls.banana = cls.create_product('Banana')
-        cls.cherry = cls.create_product('Cherry')
-        cls.damson = cls.create_product('Damson')
-        cls.elderberry = cls.create_product('Elderberry')
-        cls.fig = cls.create_product('Fig')
-        cls.grape = cls.create_product('Grape')
+    @classmethod
+    def _setup_routes(cls):
+
+        cls.picking_type_in.write({
+            'default_location_src_id': cls.env.ref('stock.stock_location_suppliers').id,
+            'default_location_dest_id': cls.recived_location.id,
+        })
+
+        cls.picking_type_internal.write({
+            'default_location_src_id': cls.recived_location.id,
```

Comment:
Not big fan of this change, internal by default should be from stock to stock.
Would be better to create putaway picking type for the inbound route.